pkg/plugin/test-keeper/plugin/comment_message.go

```diff
@@ -38,13 +42,21 @@ func CreateCommentMessage(configuration TestKeeperConfiguration, change scm.Repo
 	if configuration.LocationURL == "" {
 		msg = beginning + paragraph + noConfig
 	} else if configuration.PluginHint != "" {
-		msg = getMsgFromFile(configuration, change)
+		msg = getMsgFromConfigHint(configuration, change)
 	} else {
 		msg = getMsgWithConfigRef(configuration.LocationURL)
 	}
 	return sadIke + paragraph + msg
```

Comment:
if there is a custom message hint set, then the `sadIke` icon shouldn't be present: https://github.com/arquillian/ike-prow-plugins/pull/84#discussion_r179422668